### ISSUE TITLE
Renderer=Harfbuzz still loads the font without HarfBuzz

### ISSUE THREAD

**Issue by khaledhosny** [wspr/fontspec#396]
## Description
A sentence or two describing the issue.

## Check/indicate
- [ ] Relevant for XeTeX
- [x] Relevant for LuaTeX
- [x] Issue tracker has been searched for similar issues?
- [ ] Links to <tex.stackexchange.com> discussion if appropriate

## Minimal example demonstrating the issue
```tex
\documentclass{article}
\usepackage{fontspec}
\setmainfont{Source Han Sans SC}[Renderer=Harfbuzz]
\begin{document}
你好
\end{document}
```

## Further details

The above will still load the font without HarfBuzz when checking for font features, which 1) might give wrong results when HarfBuzz and default font loader disagree on supported font features and 2) more importantly, can freeze one’s system when loading huge fonts like this where the point of using HarfBuzz in this case would be to avoid such freeze (the default font loader consumes enormous amounts of memory the first time it tries to load a huge font like this).

**Comment by zauguin** [comment]
This should be fixed by #397.

**Comment by wspr** [comment]
Thanks Marcel, much appreciated!
